.ansible-lint

```diff
@@ -1,6 +1,7 @@
 exclude_paths:
  - ./.travis.yml
  - ./molecule/
+ - ./tasks/ssl/Linux_keystore_new.yml
```

Comment:
This is because we are using a new function to create a keystore, and ansible 2.5 does not know about this function and gives an error. So, we have no choice for testing in molecule until we use new version of ansible.